packages/email/email.js

```diff
@@ -214,7 +214,9 @@ Email.send = function (options) {
   if (options.mailComposer) {
     options = options.mailComposer.mail;
   }
-
+  if (!/^\S+@\S+$/.test(options.from)) {
```

Comment:
I think having atleast 1 character each before and after the @ should be the most basic check we should be requiring here.

Reply:
Also a good point. I'm wondering though which is more performant. Contains or regex in this situation. We only need absolute minimum check here.

Reply:
`str = '@';
console.time("test");
// console.log("String length: 100, includes:");
console.log("String length: 100, regex:");
for (let i = 0; i < 10000000; i++) {
  // typeof str == String;
  // str.includes('@');
  /^\S+@\S+$/.test(str);
}
console.timeEnd('test');`

Using this code I got these results:

1. `-> % node test.js
String length: 1, includes:
test: 4226.105ms`

2. `-> % node test.js
String length: 1, regex:
test: 369.991ms`

3. `-> % node test.js
String length: 100, includes:
test: 4787.798ms`

4. `-> % node test.js
String length: 100, regex:
test: 2500.133ms`

So, the length of the string has an adverse effect on the regex implementation, but it can perform faster for simpler strings.

Reply:
Sure, I have made the changes.

Reply:
Thanks a lot for the experience!